benchmarks/microbenchmarks/test/benchmark_config.yml

```diff
@@ -0,0 +1,20 @@
+# Sample configuration for inference kernel benchmarks
+quantization_config_recipe_names:
+  - "baseline"
+  - "int8wo"
+  - "int4wo-128"
+  - "int4wo-128-hqq"
```

Comment:
accuracy techniques like hqq shouldn't be in benchmarks. Its the same kernel as the other, just a different quant technique.